lua/modules/linear_bolzmann_solvers/response_evaluator/response_evaluator.cc

```diff
@@ -0,0 +1,127 @@
+#include "response_evaluator.h"
+#include "modules/linear_boltzmann_solvers/executors/response_evaluator.h"
+#include "lua/framework/console/console.h"
+
+namespace opensnlua::lbs::response_evaluator
```

Comment:
Fair point, however I remember us saying that the Lua layer was going to be treated differently. I am open to suggestions. I had it implemented like this before and didn't think much when copying it over. The reason I did this at first is because the `ResponseEvaluator` doesn't fit very well into the rest of the `lbs` namespace. Pretty much everything in `lbs` revolves around taking a handle to a derived `LBSSolver`, this does not. Thoughts? I am thinking that maybe 

1. `ResponseEvaluator` should be moved to a new directory called `utilities`, and
2. Maybe we introduce a namespace like `lbs_utilities` in the Lua layer.

Maybe I am overthinking it, but calling `lbs.EvaluateResponse` doesn't jive with me.